README.md

```diff
@@ -1125,8 +1125,8 @@ In addition to the ActiveRecord callbacks described above, uploaders also have c
 class MyUploader < ::CarrierWave::Uploader::Base
   before :remove, :log_removal
   private
-  def log_removal
-    ::Rails.logger.info(format('Deleting file on S3: %s', @file))
+  def log_removal(file)
+    ::Rails.logger.info(format('Deleting file on S3: %s', file))
```

Comment:
Please clarify what sort of a problem this is trying to address.

Reply:
@mshibuya 
Sorry for not explaining.

The reason I changed the argument was because an ArgmentError occurred.

```
F, [2024-01-07T18:02:54.020568 #88960] FATAL -- :
ArgumentError (wrong number of arguments (given 1, expected 0)):

app/uploaders/XXX_uploader.rb:14:in `log_retrieve_from_store'
carrierwave (2.2.2) lib/carrierwave/uploader/callbacks.rb:16:in `block in with_callbacks'
carrierwave (2.2.2) lib/carrierwave/uploader/callbacks.rb:16:in `each'
carrierwave (2.2.2) lib/carrierwave/uploader/callbacks.rb:16:in `with_callbacks'
carrierwave (2.2.2) lib/carrierwave/uploader/store.rb:86:in `retrieve_from_store!'
carrierwave (2.2.2) lib/carrierwave/uploader/versions.rb:327:in `block in retrieve_versions_from_store!'
carrierwave (2.2.2) lib/carrierwave/uploader/versions.rb:327:in `each'
carrierwave (2.2.2) lib/carrierwave/uploader/versions.rb:327:in `retrieve_versions_from_store!'
carrierwave (2.2.2) lib/carrierwave/uploader/callbacks.rb:16:in `block in with_callbacks'
carrierwave (2.2.2) lib/carrierwave/uploader/callbacks.rb:16:in `each'
```

Reply:
The retrieve_from_store callback takes an argument,
https://github.com/carrierwaveuploader/carrierwave/blob/20c6d753f8720c63718de44595351086d71b15bd/lib/carrierwave/uploader/store.rb#L104
but the remove callback doesn't.
https://github.com/carrierwaveuploader/carrierwave/blob/20c6d753f8720c63718de44595351086d71b15bd/lib/carrierwave/uploader/remove.rb#L12
So this change will break, right?

Reply:
Thanks to your explanation, I have completely understood...

I have canceled it :bow: